builder/ebs/step_prepare_fast_launch_template.go

```diff
@@ -66,7 +66,7 @@ func (s *stepPrepareFastLaunchTemplate) Run(ctx context.Context, state multistep
 	}
 
 	log.Printf("setting launch template version to %d", s.TemplateVersion)
-	state.Put("launch_template_version", s.TemplateVersion)
+	state.Put("launch_template_version", int(s.TemplateVersion))
```

Comment:
s.TemplateVersion is already of type int so there is no need to convert.
```suggestion
	state.Put("launch_template_version", s.TemplateVersion)
```